telebot/service_utils.py

```diff
@@ -0,0 +1,85 @@
+import random
+import string
+
+try:
+    # noinspection PyPackageRequirements
+    from PIL import Image
+    from io import BytesIO
+
+    pil_imported = True
+except:
```

Comment:
It should be `except ImportError` ?

Reply:
Agree, it definitely should be
That's why ctrl+C ctrl+V is bad xD